src/content/docs/zh-cn/community-resources/content.mdx

```diff
@@ -36,14 +36,18 @@ i18nReady: true
 
 ## 方案和指南
 
-查看由我们社区成员提交的向你的 Astro 项目添加功能的指导示例！
+import RecipesNav from '~/components/RecipesNav.astro';
 
-[Astro 方案](/zh-cn/recipes/) 是一系列简洁明了的操作指南，它们通过分步骤的说明，帮助读者完成特定任务。如果你想为你的 Astro 项目添加新功能或行为，方案是一个非常实用的参考资源！
+查看为 Astro 项目添加功能的指导示例。
 
-其他指南可能会解释与内容领域相关的概念，例如使用图片或处理 MDX。
+### 官方方案
+
+Astro 的官方方案是一系列简洁明了的操作指南，它们通过分步骤的说明，帮助读者完成特定任务。如果你想为你的 Astro 项目添加新功能或行为，方案是一个非常实用的参考资源！
+
+<RecipesNav />
 
 :::tip[添加你自己的内容！]
-你是否已经为 Astro 编写了一些方案？如果是的话，欢迎[编辑此页面](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx)并在相应的部分添加你的链接！
+你是否已经发布了一些使用 Astro 的方案或指南？[编辑此页面](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx) 并在下面相应的部分添加你的链接！
```

Comment:
```suggestion
你有没有发布过一些使用 Astro 的方案或指南？[编辑此页面](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx) 并在下面相应的部分添加你的链接！
```

I guess it would be more natural this way